Sprint-1/exercises/random.js

```diff
@@ -7,3 +7,11 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+console.log(num)
+
+// math.floor() = The Math.floor() function rounds down the number to the nearest whole number eg 50.78 to convert to 50.
+// math.random() = This method generates a floating-point number between 0 (inclusive) and 1 (exclusive).
+// maximum - minimum + 1 =  This expression calculates the range of possible values.
+//In this case, it calculates 100 - 1 + 1, which equals 100. This means we want to generate numbers from 1 to 100.
```

Comment:
Is this statement accurate? What is the range of numbers that can be generated by `Math.floor(Math.random() * (maximum - minimum + 1))`?

Then, what is the range of numbers we can generate after adding the `minimum`?